tfhe/examples/dist_tuniform.rs

```diff
@@ -0,0 +1,32 @@
+use tfhe::prelude::*;
+use tfhe::shortint::parameters::DynamicDistribution;
+use tfhe::{generate_keys, set_server_key, ConfigBuilder, FheUint32};
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let mut my_params = tfhe::shortint::parameters::PARAM_MESSAGE_2_CARRY_2_KS_PBS;
+    // DISCLAIMER: This is not guaranteed to be secure, thorough noise and security analysis are
+    // required by the end user
```

Comment:
Just a nitty-picky comment about this great PR.
Shouldn't we put punctuation at the end of sentences ?

Reply:
I think it's ok in the comments